app/components/common/coAuthor/index.tsx

```diff
@@ -5,6 +5,7 @@ import { trackEvent } from "../../../helpers/handleGA";
 import HIndexBox from "../hIndexBox";
 import { withStyles } from "../../../helpers/withStylesHelper";
 import Icon from "../../../icons";
+import { Tooltip } from "@material-ui/core";
```

Comment:
Please change all this import syntax to `import Tooltip  from "@material-ui/core/Tooltip";`
the above one can make Webpack do 'tree shaking' import to reduce a bundle size.

If you want to know more information, follow the below link!
https://webpack.js.org/guides/tree-shaking/